tests/test_eligibility.py

```diff
@@ -288,7 +288,7 @@ def _check_page(page, page_name, key, answer):
     return _check_page
 
 
-@unittest.mock.patch("help_to_heat.frontdoor.interface.OSApi", utils.StubAPI)
+# @unittest.mock.patch("help_to_heat.frontdoor.interface.OSApi", utils.MockApi)
```

Comment:
I commented it out as we are using mock api in real code already. I changed `StubAPI` to the new `MockApi` so that we can just uncomment this line when we switch back to real api.